resources/extruders/seckit_sktank_extruder_0.def.json

```
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "SecKit SK-Tank",
```

This should refer to the filename of the printer definition.
```suggestion
        "machine": "seckit_sktank",
```